spec/fake_braintree/transaction_spec.rb

```diff
@@ -77,6 +87,13 @@
       result = Braintree::Transaction.refund(create_id('foobar'), '1')
       expect(result).to be_success
     end
+
+    it "sets the creation time" do
+      transaction = Braintree::Transaction.refund(create_id('foobar'), '1').transaction
```

Comment:
Line is too long. [87/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.